### PR TITLE
Update collection resource

### PR DESCRIPTION
**Pull request by bwalshbmw**
Updates the collection resource to include previously undocumented params for each existing endpoint.

Collections Resource [notes](https://docs.google.com/document/d/1wECS1MjjYdrkULu6YbR3ona7Zn-wBB1oDvGe3eCu-Kg/edit#heading=h.b023mm5p85lm) from API Audit.

### CODE REVIEW

File: source/includes/_resource.collections.md [hunk 1]
```diff
@@ -151,39 +151,113 @@ $ curl -X POST "https://api.vhx.tv/collections" \
...
-      Metadata keys must be strings. There are a few reserved keys that are auto-generated, which cannot
-      be updated. For collections, the following key is reserved: <code>season_number</code>
-      <br><br>
-      Metadata values can be strings, integers, arrays, or images. An image metadata value
-      must must be a url of an image, hosted on vhx, prefixed with the text <code>image_url:</code>.
+        A set of key/value pairs that you can attach to a collection object. It can be useful for storing additional information about the collection in a structured format.
+        <br><br>
+        Metadata keys must be strings. There are a few reserved keys that are auto-generated, which cannot be updated. For collections, the following key is reserved: <code>season_number</code>
+        <br><br>
+        Metadata values can be strings, integers, arrays, or images.
```
Comment (bwalshbmw) [inline source/includes/_resource.collections.md]:
```suggestion
        Metadata values can be strings, integers, arrays, or images. An image metadata value must
        must be a url of an image, prefixed with the text <code>image_url:</code>.
```

File: source/includes/_resource.collections.md [hunk 2]
```diff
@@ -528,6 +632,86 @@ $ curl -X PUT "https://api.vhx.tv/collections/1" \
...
+    </tr>
+    <tr class="text-2 border-bottom border--light-gray">
+      <td>
+        <strong class="is-block text--navy">is_automatic</strong>
+        <span class="is-block text--transparent text-3">string</span>
+        <span class="text--transparent text-3">optional</span>
+      </td>
+      <td>
+        Set to <code>true</code> along with <code>is_featured</code> to make your featured carousel auto-scroll through the associated content. This is only available for <code>category</code> type collections.
+      </td>
```
Comment (bwalshbmw) [inline source/includes/_resource.collections.md]: `is_featured` is included twice.

File: source/includes/_resource.collections.md [hunk 3]
```diff
@@ -509,7 +613,7 @@ $ curl -X PUT "https://api.vhx.tv/collections/1" \
       must must be a url of an image, hosted on vhx, prefixed with the text <code>image_url:</code>.
```
Comment (bwalshbmw) [inline source/includes/_resource.collections.md]: Remove reference to VHX

File: source/includes/_resource.collections.md [hunk 4]
```diff
@@ -151,39 +151,113 @@ $ curl -X POST "https://api.vhx.tv/collections" \
       </td>
       <td>A publicly accessible image URL. If you prefer, you can upload images directly to a collection in the Vimeo OTT Publisher Admin.</td>
     </tr>
-    <tr class="is-internal text-2 border-bottom border--light-gray">
+    <tr class="text-2 border-bottom border--light-gray">
       <td>
         <strong class="is-block text--navy">metadata</strong>
```
Comment (bwalshbmw) [inline source/includes/_resource.collections.md]: General comment that metadata should be moved down in the argument lists as standard metadata is the preferred option at this point. In fact, should we even be including this param?

File: source/includes/_resource.collections.md [hunk 5]
```diff
@@ -367,6 +441,26 @@ $ curl -X GET -G "https://api.vhx.tv/collections" \
...
       <td>The sort to order the results. Options are <code>alphabetical</code>, <code>newest</code>, <code>oldest</code>, or <code>latest</code>.</td>
     </tr>
+    <tr class="text-2 border-bottom border--light-gray">
+      <td>
+        <strong class="is-block text--navy">active</strong>
+        <span class="is-block text--transparent text-3">string</span>
+        <span class="text--transparent text-3">optional</span>
+      </td>
+      <td>
+        Set this to <code>true</code> to only return active collections. Set to <code>false</code> to only return inactive collections.
```
Comment (bwalshbmw) [inline source/includes/_resource.collections.md]: Could better explain what active means in this context.

File: source/includes/_resource.collections.md [hunk 6]
```diff
@@ -528,6 +632,86 @@ $ curl -X PUT "https://api.vhx.tv/collections/1" \
...
+      </td>
+    </tr>
+    <tr class="text-2 border-bottom border--light-gray">
+      <td>
+        <strong class="is-block text--navy">tags</strong>
+        <span class="is-block text--transparent text-3">array of strings</span>
+        <span class="text--transparent text-3">optional</span>
+      </td>
+      <td>
+        Tags that you want associated in the system with your video. These can be used to help find your collection.
```
Comment (bwalshbmw) [inline source/includes/_resource.collections.md]:
```suggestion
        Tags that you want associated in the system with your collection. These can be used to help find your collection.
```

Comment (bwalshbmw) [inline source/includes/_resource.collections.md]: Revisit this and explain better.

File: source/includes/_resource.collections.md [hunk 7]
```diff
@@ -528,6 +616,102 @@ $ curl -X PUT "https://api.vhx.tv/collections/1" \
...
+      </td>
+    </tr>
+    <tr class="text-2 border-bottom border--light-gray">
+      <td>
+        <strong class="is-block text--navy">tags</strong>
+        <span class="is-block text--transparent text-3">array of strings</span>
+        <span class="text--transparent text-3">optional</span>
+      </td>
+      <td>
+        Tags that you want associated in the system with the collection.
```
Comment (bwalshbmw) [inline source/includes/_resource.collections.md]: Need to include that you can separate multiple entries with commas

**Review by kaitlinwiener — APPROVED** [review]
💯

**Pull request merged**